src/components/calcite-block/calcite-block.tsx

```diff
@@ -1,8 +1,10 @@
-import { Component, Element, Event, EventEmitter, Prop, h } from "@stencil/core";
+import { Component, Element, Event, EventEmitter, Host, Prop, h } from "@stencil/core";
 import { chevronDown16, chevronUp16 } from "@esri/calcite-ui-icons";
 import { CSS, TEXT } from "./resources";
 import CalciteIcon from "../utils/CalciteIcon";
 import { CalciteTheme } from "../interfaces";
+import { VNode } from "@esri/calcite-components/dist/types/stencil.core";
```

Comment:
[fixme] Change to `import { VNode } from "@stencil/core/dist/declarations";`

Reply:
Please have another peek.